docs/extras/styling.md

```diff
@@ -150,3 +150,79 @@ The `display_port_labels` render parameter allows you to display the port labels
         <img src="../../assets/images/ContextDiagram of Hierarchy display_port_labels.svg" width="1000000">
         <figcaption>Context diagram of Hierarchy LogicalComponenet with type [LAB] display_port_labels</figcaption>
     </figure>
+
+# PVMT Styling
+
+The `pvmt_styling` render parameter allows you to dynamically style diagram elements based on PVMT (Property Value Management Tool) properties attached to model elements in Capella. This enables custom coloring and styling based on model metadata.
+
+!!! info "PVMT Integration"
+
+    This feature requires PVMT properties to be defined in your Capella model. The styling system reads color values from specific PVMT properties and applies them to diagram elements.
+
+## Configuration
+
+The `pvmt_styling` parameter accepts a dictionary with the following structure:
+
+``` yaml
+pvmt_styling:
+  value_groups:
+    - "Test.Kind.Color"  # List of PVMT value group names
+  children_coloring: false  # Whether to apply styling to child elements
+```
```

Comment:
For the example here, I think it would make more sense to use Python syntax. All the other examples on this page are Python snippets, so this one using YAML would be the odd one out.